TFT/src/User/API/RRFStatusControl.c

```diff
@@ -0,0 +1,71 @@
+#include "RRFStatusControl.h"
+#include "includes.h"
+
+#define RRF_NORMAL_STATUS_QUERY_MS 1000
+#define RRF_FAST_STATUS_QUERY_MS 500
+
+// available status: status:
+// I=idle, P=printing from SD card, S=stopped (i.e. needs a reset), C=running config file (i.e starting up),
+// A=paused, D=pausing, R=resuming from a pause, B=busy (e.g. running a macro), F=performing firmware update
+static char rrf_status = 'I';
+
+static uint16_t rrf_query_interval = RRF_NORMAL_STATUS_QUERY_MS;
+static bool macro_busy = false;
+
+void rrfStatusSet(char status)
+{
+  rrf_status = status;
+  if (status != 'B')
+  {
+    macro_busy = false;
+  }
+}
+
+void rrfStatusSetBusy(void)
```

Comment:
make all of those functions made by only one line as staic inline. E.g. "static inline void rrFStatusSet etc...) it is more efficient and uses les flash space (in that case)